frontend/js/src/user-feed/UserFeed.tsx

```diff
@@ -625,13 +624,6 @@ export default function UserFeedPage() {
           <UserSocialNetwork user={currentUser} />
         </div>
       </div>
-      <BrainzPlayer
```

Comment:
No, the implementation quite similar to how we do on other components. But since we're using infiniteQuery, need to take care of duplications when more events are fetched.
I noticed this while testing, so I included this in the doc so that I don't forget about this.

Reply:
Yep, I think it's fine for v1